src/js/components/Poll/PublicRegisterModal.vue

```diff
@@ -139,17 +139,17 @@ export default {
 				}
 			} else if (this.userName.length < 3) {
 				return {
-					result: t('polls', 'Please use at least 3 characters.'),
+					result: t('polls', 'Username must be at least 3 characters.'),
 					status: 'error',
 				}
 			} else if (!this.isValidName) {
 				return {
-					result: t('polls', 'Invalid name'),
+					result: t('polls', 'Invalid username'),
```

Comment:
We avoided the usage of the term username, because some users were confused and entered their username, which they use to login and this lead to a validation error.

Reply:
This in such a case should also be changed in other text strings, such as:
https://github.com/nextcloud/polls/blob/ad6e46e17406f12e4a8f6c1443841dbd0108d7b5/src/js/components/Poll/PublicRegisterModal.vue#L142

Reply:
This in such a case should also be changed in other text strings, such as:
https://github.com/nextcloud/polls/blob/ad6e46e17406f12e4a8f6c1443841dbd0108d7b5/src/js/components/Poll/PublicRegisterModal.vue#L137
e.t.c.

Reply:
Absolutely. Would you mind correcting this, too?

Reply:
Changed 😉. Can you approve?